### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by icepuma** [watercooler-labs/toggl-cli#68] [General]
Hey,
want to contribute @ [toggl-rs](https://crates.io/crates/toggl-rs)?

My main goal is to have all models for API v9 to use it @ [fbtoggl](https://github.com/icepuma/fbtoggl).

Best regards
icepuma

**Comment by shantanuraj** [comment]
Hey @icepuma 
I saw `toggl-rs` and `fbtoggl` great work there, they are very comprehensive.

If you have a list of models that you wish to add to toggl-rs I could look into it.
Would be happy to help you.

**Comment by icepuma** [comment]
I will write up a contribution guide this week and then start working on it again. We then can both use the lib in our clients :)
